nemo/lightning/io/mixin.py

```diff
@@ -326,6 +326,7 @@ def _get_connector(cls, ext, path=None, importer=True, **kwargs) -> ModelConnect
                         when required.
         """
         _path = None
+        ext = str(ext)
```

Comment:
Docstring already suggests this should be a string, but no harm in this I guess. Can we add type hints for the args as well?

Reply:
Added

Reply:
sorry for nitpick, can you update docstring type of `path` to match signature typehint (or vice versa), for consistency.

Reply:
Done~